user_guide_src/source/installation/running.rst

```diff
@@ -45,15 +45,15 @@ Initial Configuration
         run using the "production" environment. See also :ref:`environment-constant`.
 
 .. note:: If you will be running your site using a web server (e.g., Apache or Nginx),
-    you will need to modify the permissions for the ``writable`` folder inside
+    you will need to modify the permissions for the **writable** folder inside
     your project, so that it is writable by the user or account used by your
     web server.
 
 Local Development Server
 ========================
 
 CodeIgniter 4 comes with a local development server, leveraging PHP's built-in web server
-with CodeIgniter routing. You can use the ``serve`` script to launch it,
+with CodeIgniter routing. You can use the ``spark serve`` command to launch it,
 with the following command line in the main directory::
```

Comment:
It doesn't sound good when you read the whole sentence. Maybe something simpler?
```suggestion
with CodeIgniter routing. You can launch it, with the following command line 
in the main directory::
```

Reply:
Thank you.